src/services/translations/index.ts

```diff
@@ -0,0 +1,64 @@
+import LocalizedStrings from 'react-localization';
+
+import nb from '../../l10n/nb.json';
+import nn from '../../l10n/nn.json';
+import en from '../../l10n/en.json';
+
+import type { ChangeLanguageCallback, FormatObject, Formatted } from './types';
+
+import { Language } from './enums';
+
+class TranslationsService {
+  private language: Language;
+
+  private changeLanguageCallback?: ChangeLanguageCallback;
+
+  private readonly translations = new LocalizedStrings({
+    [Language.NB]: nb,
+    [Language.NN]: nn,
+    [Language.EN]: en
+  });
+
+  public async init(
+    language: Language,
+    callback?: ChangeLanguageCallback
+  ): Promise<void> {
+    this.language = language;
+    this.changeLanguageCallback = callback;
+
+    this.translations.setLanguage(this.language);
+  }
+
+  public getLanguage(): Language {
+    return this.language;
+  }
+
+  public translate<T extends Formatted>(
+    key: string,
+    values?: FormatObject<T>
+  ): Array<string | T> | string {
```

Comment:
Vil en slik forenkling fungere?

```suggestion
  public translate(
    key: string,
    values?: FormatObject
  ): string | Fomatted[] {
```